src/components/LabelEdit.vue

```diff
@@ -165,7 +165,7 @@ export default {
 
 <template>
   <v-list-item dense class="px-0">
-    <v-list-item-content width="800px" class="overflow-x-scroll">
+    <v-list-item-content width="800px" class="overflow-x-scroll scroll-across">
```

Comment:
I'm not sure `overflow-x-scroll` exists in our stylesheet (it should); if it did, wouldn't this just need the 1 class?

Reply:
It didn't but it does now - good call.